rosdep/python.yaml

```diff
@@ -3340,6 +3351,9 @@ python-xlib:
   fedora: [python-xlib]
   gentoo: [dev-python/python-xlib]
   ubuntu: [python-xlib]
+python-xmltodict:
+  ubuntu:
+    xenial: [python-xmltodict]
```

Comment:
This package is available on all ubuntu platform starting with xenial, please add them all. We also usually add the debian key as well